tensorflow/core/profiler/internal/cpu/host_tracer_test.cc

```diff
@@ -107,7 +109,7 @@ TEST(HostTracerTest, CollectsTraceMeEventsAsRunMetadata) {
 }
 
 TEST(HostTracerTest, CollectsTraceMeEventsAsXSpace) {
-  int32 thread_id;
+  std::atomic<int32> thread_id;
```

Comment:
Now that this variable is atomic, replace the use of the assignment (operator=) below to store().
That's customary within Google to make it explicit we are interacting with an atomic variable.